service/src/test/java/bio/terra/cbas/dependencies/leonardo/TestAppUtils.java

```diff
@@ -434,5 +465,77 @@ public TestAppUtils() throws IOException {
                     "labels": {}
                 }""",
                 Map.of("workspaceId", workspaceId)));
+
+    cromwellRunnerAppResponse =
+        ListAppResponse.fromJson(
+            StringSubstitutor.replace(
+                """
+                {
+                    "workspaceId": "${workspaceId}",
+                    "cloudContext": {
+                        "cloudProvider": "AZURE",
+                        "cloudResource": "blah-blah-blah"
+                    },
+                    "kubernetesRuntimeConfig": {
+                        "numNodes": 1,
+                        "machineType": "Standard_A2_v2",
+                        "autoscalingEnabled": false
+                    },
+                    "errors": [],
+                    "status": "RUNNING",
+                    "proxyUrls": {
+                        "cbas": "https://lzblahblahblah.servicebus.windows.net/cromwell-runner-app-${workspaceId}/cbas",
+                        "cbas-ui": "https://lzblahblahblah.servicebus.windows.net/cromwell-runner-app-${workspaceId}/",
+                        "cromwell": "https://lzblahblahblah.servicebus.windows.net/cromwell-runner-app-${workspaceId}/cromwell"
+                    },
+                    "appName": "cromwell-runner-app-${workspaceId}",
+                    "appType": "CROMWELL_RUNNER_APP",
+                    "diskName": null,
+                    "auditInfo": {
+                        "creator": "me@broadinstitute.org",
+                        "createdDate": "2023-02-09T16:01:36.660590Z",
+                        "destroyedDate": null,
+                        "dateAccessed": "2023-02-09T16:01:36.660590Z"
+                    },
+                    "accessScope": null,
+                    "labels": {}
+                }""",
+                Map.of("workspaceId", workspaceId)));
+
+    workflowsAppResponse =
+        ListAppResponse.fromJson(
+            StringSubstitutor.replace(
+                """
+                {
+                    "workspaceId": "${workspaceId}",
+                    "cloudContext": {
+                        "cloudProvider": "AZURE",
+                        "cloudResource": "blah-blah-blah"
+                    },
+                    "kubernetesRuntimeConfig": {
+                        "numNodes": 1,
+                        "machineType": "Standard_A2_v2",
+                        "autoscalingEnabled": false
+                    },
+                    "errors": [],
+                    "status": "RUNNING",
+                    "proxyUrls": {
+                        "cbas": "https://lzblahblahblah.servicebus.windows.net/terra-app-${workspaceId}/cbas",
+                        "cbas-ui": "https://lzblahblahblah.servicebus.windows.net/terra-app-${workspaceId}/",
```

Comment:
Similar note to above, workflows app won't have cbas-ui and will have cromwell-reader instead of cromwell. Both will also have listener proxy URLs but I don't think those are necessary to include here. Workflows app will also have `"accessScope": "WORKSPACE_SHARED"`